internal/api/tests/private/highLevelConnectionStatus_test.go

```diff
@@ -25,7 +25,7 @@ func getConnectionStatus(payload ApiInternalHighlevelConnectionStatusJSONRequest
 	Expect(err).ToNot(HaveOccurred())
 	Expect(res.StatusCode()).To(Equal(http.StatusOK))
 
-	return res.JSON200, res
+	return resp, res
```

Comment:
I would modify this to return `res` and `err` instead of `resp` and `res`.

The caller can then check to make sure that an error did not occur and the caller can use either JSON200 or JSON400 (whichever is appropriate).